docs/docs/installation/8-nocode-iot-deployment.md

```diff
@@ -0,0 +1,45 @@
+# No-Code IoT Deployment
+
+We built a tool for deploying your Groundlight detector on any supported ESP32 Camera Board without writing any code. This tool is available at https://code.groundlight.ai/groundlight-embedded-uploader/.
+
+## Easy Deployment
+
+This tool is designed to make it as easy as possible to deploy your Groundlight detector on an ESP32 Camera Board. You can deploy your detector in just a few clicks.
+
+1. Go to https://code.groundlight.ai/groundlight-embedded-uploader/.
+2. Plug your ESP32 Camera Board into your computer with a USB cable.
+3. Click through the steps to upload your detector to your ESP32 Camera Board.
+
+## Notification Options
+
+The tool supports the following notification options for your deployed detector:
+
+- Email
+- SMS (With Twilio)
+- Slack
+
+## Multiple Supported Boards
+
+The supported boards are:
```

Comment:
tested with the following boards  many similar boards should work as long as they have xxxx  but may require building from source and changing IO pin configurations. 

should be any of the ESP32CAM boards not just AI Thinker ?  might be helpful to include a picture of the boards 

also strip the extra bits from the amazon link .

Reply:
Links fixed, added images, better preface as suggested